src/main/java/be/jidoka/jdk/keycloak/admin/domain/HasUserPersonalData.java

```diff
@@ -0,0 +1,12 @@
+package be.jidoka.jdk.keycloak.admin.domain;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+public interface HasUserPersonalData {
```

Comment:
UpdateUserPersonalData?

Reply:
Done updated to UserPersonalDataCommand.